### DISCUSSION TITLE
使用volantis 的时候 为什么上传图片默认不会转换成base64啊

### DISCUSSION THREAD

**Discussion by TipsyDr** [walinejs/waline#811] [Q&A] [answered]
使用volantis 的时候 为什么上传图片默认不会转换成base64啊

**Comment by lizheming** [comment, marked as answer]
https://github.com/volantis-x/hexo-theme-volantis/blob/dd7a683818abf94a352f39bdfa8d04f16c8a130c/layout/_plugins/comments/waline/script.ejs#L15-L22 看了下，应该是主题自定义了 uploadImage 方法了，并暴露了个 imageHosting 的变量

**Reply by TipsyDr** [reply]
了解了 感谢回复

**Reply by lizheming** [reply]
7bu 图床应该不免费使用了，所以可能需要改下主题，把主题的这个配置去掉就可以了
